src/transformers/tokenization_bart.py

```diff
@@ -135,44 +194,45 @@ def prepare_translation_batch(
         tgt_texts: Optional[List[str]] = None,
         tgt_lang: str = "ro_RO",
         max_length: Optional[int] = None,
-        pad_to_max_length: bool = True,
+        padding: str = "longest",
```

Comment:
This is a breaking change; is `prepare_translation_batch` user-facing? If so, can we keep backwards compatibility by still handling `pad_to_max_length` with a `FutureWarning`?

Reply:
sure, good idea!